drivers/led_strip/ws2812_i2s.c

```diff
@@ -69,6 +69,8 @@ static int ws2812_strip_update_rgb(const struct device *dev, struct led_rgb *pix
 	const uint8_t sym_one = cfg->nibble_one;
 	const uint8_t sym_zero = cfg->nibble_zero;
 	const uint32_t reset_word = cfg->active_low ? ~0 : 0;
+	const uint16_t pre_delay_words =
+		cfg->active_low ? cfg->reset_words : WS2812_I2S_PRE_DELAY_WORDS_DEFAULT;
```

Comment:
Note that having a signal active at idle state can also happen with a signal active high.

Reply:
True. It should probably have its own property.